src/main/java/no/nav/pto/veilarbportefolje/opensearch/OpensearchService.java

```diff
@@ -49,6 +50,12 @@ public BrukereMedAntall hentBrukere(
             Integer antall,
             BrukerinnsynTilgangFilterType brukerinnsynTilgangFilterType
     ) {
+        BrukerinnsynTilganger brukerInnsynTilganger = authService.hentVeilederBrukerInnsynTilganger();
+
+        if (brukerInnsynTilganger.harAlle() && BrukerinnsynTilgangFilterType.BRUKERE_SOM_VEILEDER_IKKE_HAR_INNSYNSRETT_PÅ == brukerinnsynTilgangFilterType) {
```

Comment:
I think we should never allow to have `BRUKERE_SOM_VEILEDER_IKKE_HAR_INNSYNSRETT_PÅ` in this method